src/services/api/handlers/packageActions.ts

```diff
@@ -212,6 +212,7 @@ export async function respondToRai(body: RaiResponse, rais: any) {
     await pool.close();
   }
   console.log("heyo");
```

Comment:
?

Reply:
@kevinhaube This must have been from a merge from master.

Reply:
@mdial89f i see you made this update last month. any thought on this?